src/rgw/rgw_tag_s3.cc

```diff
@@ -30,7 +30,7 @@ void RGWObjTagEntry_S3::dump_xml(Formatter *f) const {
 void RGWObjTagSet_S3::decode_xml(XMLObj *obj) {
   vector<RGWObjTagEntry_S3> entries;
 
-  RGWXMLDecoder::decode_xml("Tag", entries, obj, true);
+  RGWXMLDecoder::decode_xml("Tag", entries, obj, false);
```

Comment:
it's not clear what that last boolean flag means, can you say?

Reply:
done